nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -398,7 +398,7 @@ def start_server_app(self):
             False,
             None,
         )
-        server_app_runner = SimulatorServerAppRunner()
+        server_app_runner = SimulatorServerAppRunner(self.services)
```

Comment:
since in this class `self.services` just mean 1 FL server, I prefer we rename it to self.server or self.fl_server something like that, but that can be done in a separate PR

Reply:
Done.